missile_toad/src/core/game_descriptor.cpp

```diff
@@ -31,6 +31,7 @@ std::optional<missiletoad::core::GameDescriptor> missiletoad::core::load_game_de
     auto game_descriptor = missiletoad::core::GameDescriptor();
 
     game_descriptor.name = document["name"].GetString();
```

Comment:
This needs to be formatted, please check the following [document](https://github.com/Alan5142/missile-toad/blob/main/docs/Building.md#formatting) to get instructions on how to format the code. You might need LLVM tools if they're not installed. 

pre-commit hooks should do this automatically after each commit, check the instructions [here](https://github.com/Alan5142/missile-toad/blob/main/docs/Building.md#committing)